Dockerfile

```diff
@@ -0,0 +1,87 @@
+# ------------------
+# New tmp image
+# ------------------
+FROM node:18.19.1-bullseye-slim AS tmp
```

Comment:
nitpick: tmp being a overused term, what about calling the initial image `builder` or something like this

Reply:
sure good point